src/wallet/coin_selection.rs

```diff
@@ -1505,12 +1480,11 @@ mod test {
             utxos,
             0,
             target_amount as i64,
-            FEE_AMOUNT,
```

Comment:
In all the other tests you do `target_amount = *whatever it was before* + FEE_AMOUNT", maybe you should do the same here?

Reply:
Sure! Although, now that I review the test again, it's impossible for it to fail :eyes:.

Reply:
Yeah, you can't do much with unit testing on the SDR, other than hoping it doesn't panic